ROADMAP.md

```diff
@@ -0,0 +1,104 @@
+# `pflag` roadmap
+
+## Beef up the team of maintainers
+
+3 to 5 volunteers. No less, no more.
+
+Would be nice to open a slack or discord channel to make debate more lively.
+Otherwise, we may use github discussions.
+
+## Agree on the [CONTRIBUTING guidelines](./CONTRIBUTING.md)
+
+There are quite a few debatable things there: let's find a minimal consensus
+(possibly defer decisions about some details to a later stage).
+
+## Agree on a general direction
+
+Proposed principles:
+
+* promise of compatibility (i.e. no breaking `v2` ever)
+
+> Example: solving issue #384 by adding support for a default value would break compatibility.
+> If we want to solve this issue, we need a new API.
+
+* prioritize use cases from `cobra` & `viper`, acknowledged as the main venues for importing `pflag` indirectly
+
+> Example: solving issue #250 typically comes from a `cobra` user.
+
+* prioritize code simplification & maintenance reduction over new features
+
+> Examples: PRs #235, #248, #332, #369, #378, #380 are representative of that effort
+
+* deflect stream of innovation, desire to share new ideas to some dedicated `contrib` repo
+
+> Examples: issue #246 is typically such a new flag type, requiring a specific import.
+
+
+## Issue triage
+
+There are 88 outstanding issues. Let's triage them first.
+
+For every one of them:
+  * categorize: bug report, feature request/enhancement, code quality/testing, extensions ... (add new labels)
+  * post a comment/reply to the OP
+
+Proposed issue labels:
+* bug
```

Comment:
I usually like prefixing labels with things like `kind/`, `resolution/`, etc. It makes distinguishing between different labeling schemes easier.

Reply:
ok good point